miden/src/examples/mod.rs

```diff
@@ -140,14 +140,17 @@ where
 
     assert_eq!(
         expected_result,
-        outputs.stack_outputs(num_outputs),
+        outputs.stack_truncated(num_outputs),
         "Program result was computed incorrectly"
     );
 
+    let kernel = miden::Kernel::default();
+    let program_info = ProgramInfo::new(program.hash(), kernel);
+
     if fail {
         outputs.stack_mut()[0] += 1;
-        assert!(miden::verify(program.hash(), stack_inputs, &outputs, proof).is_err())
+        assert!(miden::verify(program_info, stack_inputs, outputs, proof).is_err())
     } else {
-        assert!(miden::verify(program.hash(), stack_inputs, &outputs, proof).is_ok());
+        assert!(miden::verify(program_info, stack_inputs, outputs, proof).is_ok());
     }
```

Comment:
Question: How was this working (or even compiling) before this change?

Reply:
It's because of this issue here:

https://github.com/0xPolygonMiden/miden-vm/issues/702

It's a test inside of an example (not very obvious to check). I can't fix it now because I'm facing issues with my local GH client that is refusing to update the CI